pootle/apps/pootle_misc/checks.py

```diff
@@ -266,56 +267,82 @@ class SkipCheck(Exception):
     pass
 
 
-class ENChecker(checks.TranslationChecker):
+class ENChecker(checks.UnitChecker):
+
+    def __init__(self, checkerconfig=None, excludefilters=None,
```

Comment:
Thank you for spotting this — it was a leftover of the initial customizations I made on top of `TranslationChecker`.